src/msgwindow.c

```diff
@@ -262,19 +305,13 @@ static void prepare_compiler_tree_view(void)
 	/*g_signal_connect(selection, "changed", G_CALLBACK(on_msg_tree_selection_changed), NULL);*/
 }
 
-
-static const GdkColor color_error = {0, 65535, 0, 0};
-
 static const GdkColor *get_color(gint msg_color)
 {
-	static const GdkColor dark_red = {0, 65535 / 2, 0, 0};
-	static const GdkColor blue = {0, 0, 0, 0xD000};	/* not too bright ;-) */
-
 	switch (msg_color)
 	{
 		case COLOR_RED: return &color_error;
-		case COLOR_DARK_RED: return &dark_red;
-		case COLOR_BLUE: return &blue;
+		case COLOR_DARK_RED: return &color_context;
+		case COLOR_BLUE: return &color_message;
```

Comment:
We really should rename those color names someday to something semantic.  But well.